lib/ansible/plugins/action/__init__.py

```diff
@@ -890,12 +890,16 @@ def _parse_returned_data(self, res):
                 if res['stderr'].startswith(u'Traceback'):
                     data['exception'] = res['stderr']
 
+            # The default
+            data['msg'] = "MODULE FAILURE"
+
             # try to figure out if we are missing interpreter
```

Comment:
It's better to put this comment above setting the default value as the default value is part of the section of code  which tries "to figure out if we are missing interpreter".